docs/src/test-advanced-js.md

```diff
@@ -301,6 +301,56 @@ test('test', async ({ page }) => {
 });
 ```
 
+You can make arbitary data available in your tests from your global setup file by attaching it to `process.env`.
+
```

Comment:
Inside line 214 let's replace 'environment' with 'environment variables' to make it more clear.